cmd/nerdctl/pull.go

```diff
@@ -120,73 +120,27 @@ func pullAction(cmd *cobra.Command, args []string) error {
 }
 
 func ensureImage(ctx context.Context, cmd *cobra.Command, client *containerd.Client, globalOptions types.GlobalCommandOptions, rawRef string, ocispecPlatforms []v1.Platform, pull string, unpack *bool, quiet bool) (*imgutil.EnsuredImage, error) {
```

Comment:
this function should be removed to pkg package

Reply:
Seems that the function is only used by the run command. Maybe better to move to pkg/cmd/run IMO

Reply:
Moved to run. Done.